azure-mgmt-website/src/main/java/com/microsoft/azure/management/website/WebApp.java

```diff
@@ -194,7 +168,7 @@
      */
     String defaultHostName();
 
-    List<HostNameBinding> hostNameBindings() throws CloudException, IOException;
+    List<HostNameBinding> getHostNameBindings();
```

Comment:
is there a unique string that could be used as the index if this returned Map<HostNameBinding> instead of List?